pkgs/applications/video/subtitleeditor/default.nix

```diff
@@ -35,20 +38,11 @@ stdenv.mkDerivation rec {
     isocodes
   ];
 
-  NIX_CFLAGS_COMPILE = "-std=c++11 -DDEBUG";
-
   enableParallelBuilding = true;
 
-  doCheck = true;
```

Comment:
Because there is a failure when making check in the `po` subdirectory:
```
[...]
Making check in po
make[1]: Entering directory '/tmp/nix-build-subtitle-editor-0.53.0.drv-18/subtitleeditor-0.53.0/po'
INTLTOOL_EXTRACT="/nix/store/cl7za9xhla6nax0yhg37q10c12pw30wj-intltool-0.51.0/bin/intltool-extract" XGETTEXT="/nix/store/mlqqfj2b2y42p2fnj05m5706b7qg3vab-gettext-0.19.8/bin/xgettext" srcdir=. /nix/store/cl7za9xhla6nax0yhg37q10c12pw30wj-intltool-0.51.0/bin/intltool-update --gettext-package subtitleeditor --pot
rm -f missing notexist
srcdir=. /nix/store/cl7za9xhla6nax0yhg37q10c12pw30wj-intltool-0.51.0/bin/intltool-update -m
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\${ <-- HERE ?PACKAGE_NAME}?/ at /nix/store/cl7za9xhla6nax0yhg37q10c12pw30wj-intltool-0.51.0/bin/intltool-update line 1071, <CONF> line 417.
The following files contain translations and are currently not in use. Please
consider adding these to the POTFILES.in file, located in the po/ directory.

plugins/actions/template/dialog-template-save-as.ui
plugins/actions/template/template.cc

If some of these files are left out on purpose then please add them to
POTFILES.skip instead of POTFILES.in. A file 'missing' containing this list
of left out files has been written in the current directory.
if [ -r missing -o -r notexist ]; then \
  exit 1; \
fi
make[1]: *** [Makefile:495: check] Error 1
make[1]: Leaving directory '/tmp/nix-build-subtitle-editor-0.53.0.drv-18/subtitleeditor-0.53.0/po'
make: *** [Makefile:477: check-recursive] Error 1
[...]
```